include/readdy/model/observables/ReactionCounts.h

```diff
@@ -33,13 +33,21 @@
 #pragma once
 
 #include "Observable.h"
+#include <readdy/common/ParticleTypeTuple.h>
+#include <readdy/model/Particle.h>
+#include <readdy/model/KernelContext.h>
 
 NAMESPACE_BEGIN(readdy)
 NAMESPACE_BEGIN(model)
 NAMESPACE_BEGIN(observables)
 
-class ReactionCounts : public Observable<std::tuple<std::vector<std::size_t>, std::vector<std::size_t>>> {
+class ReactionCounts : public Observable<std::pair<
+        std::unordered_map<readdy::model::Particle::type_type, std::vector<std::size_t>>,
```

Comment:
`readdy::model::Particle::type_type` is `particle_type_type`, which can be found in common.h - then the include of Particle.h is not necessary.